projects/components/src/modal/modal.ts

```diff
@@ -13,6 +13,7 @@ export interface ModalConfig<TData = unknown> {
 export const enum ModalSize {
   Small = 'small',
   Medium = 'medium',
+  LargeShort = 'large-short',
```

Comment:
hah... I don't get why we bother

Reply:
Yeah...

I almost think we should have size inputs for height and width. Then we can just mix and match. It will keep us semi-consistent at least.